src/app/pages/system/advanced/advanced-settings.component.ts

```diff
@@ -0,0 +1,316 @@
+import { ConsoleFormComponent } from './console-form/console-form.component'
+import { TunableFormComponent } from './../tunable/tunable-form/tunable-form.component'
+import {
+  Component,
+  OnInit,
+  OnDestroy,
+  Injector,
+  ApplicationRef,
+} from '@angular/core'
+import {
+  WebSocketService,
+  SystemGeneralService,
+  DialogService,
+  LanguageService,
+  StorageService,
+} from '../../../services'
+import { CoreService, CoreEvent } from 'app/core/services/core.service'
+import { LocaleService } from '../../../services/locale.service'
+import { ModalService } from '../../../services/modal.service'
+import { MatDialog } from '@angular/material/dialog'
+import { Router, ActivatedRoute } from '@angular/router'
+import { HttpClient } from '@angular/common/http'
+import { helptext_system_general as helptext } from 'app/helptext/system/general'
+import { helptext_system_advanced as helptext_advanced } from 'app/helptext/system/advanced'
+import { AppLoaderService } from '../../../services/app-loader/app-loader.service'
+import { Subscription, Subject } from 'rxjs'
+import { T } from 'app/translate-marker'
+import { KernelFormComponent } from './kernel-form/kernel-form.component'
+import { SyslogFormComponent } from './syslog-form/syslog-form.component'
+import { EmptyType } from 'app/pages/common/entity/entity-empty/entity-empty.component'
+
+@Component({
+  selector: 'app-advanced-settings',
+  templateUrl: './advanced-settings.component.html',
+})
+export class AdvancedSettingsComponent implements OnInit, OnDestroy {
+  dataCards = []
+  sysctlTitle = helptext_advanced.fieldset_sysctl
+  configData: any
+  refreshCardData: Subscription
+  displayedColumns: any
+  dataSource: any[] = []
+  refreshTable: Subscription
+  getSysctlSubscription: Subscription
+  getAdvancedConfig: Subscription
+  getDatasetConfig: Subscription
+  public formEvents: Subject<CoreEvent>
+  syslog: boolean
+  entityForm: any
+
+  // Components included in this dashboard
+  protected tunableFormComponent = new TunableFormComponent(
+    this.router,
+    this.route,
+    this.ws,
+    this.injector,
+    this.appRef,
+    this.sysGeneralService
+  )
+  
+  protected consoleFormComponent = new ConsoleFormComponent(
+    this.router,
+    this.language,
+    this.ws,
+    this.dialog,
+    this.loader,
+    this.http,
+    this.storage,
+    this.sysGeneralService,
+    this.modalService
+  )
+  
+  protected kernelFormComponent = new KernelFormComponent(
+    this.router,
+    this.language,
+    this.ws,
+    this.dialog,
+    this.loader,
+    this.http,
+    this.storage,
+    this.sysGeneralService,
+    this.modalService
+  )
+  
+  protected syslogFormComponent = new SyslogFormComponent(
+    this.router,
+    this.language,
+    this.ws,
+    this.dialog,
+    this.loader,
+    this.http,
+    this.storage,
+    this.sysGeneralService,
+    this.modalService
+  )
+  
+  public emptyPageConf: EmptyConfig = {
+    type: EmptyType.loading,
```

Comment:
Let's handle the no_page_data case here instead of loading.
We can also add the following
Title: “No tunables configured”
Message: “To configure sysctls, click the ‘Add’ button”